storage/manager.go

```diff
@@ -130,11 +134,11 @@ func (m *Manager) PumpFn(name string, pool engine.Engine, threshold int64) func(
 			return false
 		}
 		metrics.storageDelJobs.WithLabelValues(name).Add(float64(len(jobsID)))
-		return len(jobsID) == maxJobBatchSize
+		return int64(len(jobsID)) == batchSize
 	}
 }
 
-func (m *Manager) AddPool(name string, pool engine.Engine, threshold int64) {
+func (m *Manager) AddPool(name string, pool engine.Engine, threshold int64, maxPumpBatchSize int64) {
```

Comment:
Remove this parameter. The manager can store the config when creating, then get this batch size from the config.

Reply:
Just updated, please take another look.